shaders/gshader_to_mshader/gshader_to_mshader.frag

```diff
@@ -25,8 +25,8 @@ layout (location = 0) out vec4 outFragColor;
 
 void main() 
 {
-	float ambientStrength = 0.2;
-	vec3 ambient = ambientStrength * vec3(1.0, 1.0, 1.0);
+	float ambientStrength_f = 0.2;
```

Comment:
Ahm... I did not ask to add an f to the variable names, but to the constants: `0.2` - > `0.2f` etc.

Reply:
fixed

Reply:
I haven't seen that much in shaders for other samples but done.